examples/nextjs/components/Web3Provider.tsx

```diff
@@ -13,11 +17,17 @@ const config = createConfig(
 
 const queryClient = new QueryClient();
 
+const options: DaimoPayOptions = {
+  closeOnSuccess: true,
+};
+
 export const Web3Provider = ({ children }: { children: React.ReactNode }) => {
   return (
     <WagmiProvider config={config}>
       <QueryClientProvider client={queryClient}>
-        <DaimoPayProvider debugMode>{children}</DaimoPayProvider>
+        <DaimoPayProvider debugMode options={options}>
```

Comment:
I believe it’s bad practice to have state like this split between the provider and the button itself.

Since theme, payID etc are all attached to the button itself, I would suggest moving it there

Reply:
hmm, it looks connectkit already puts a lot of options into the provider?

reducedMotion, hideTooltips etc

i can try to move `closeOnSuccess` to the DaimoPayButton if that's better. as far as i can tell, they are putting stuff into the provider that would be the same across all buttons in the case where there's more than one pay/connectkit button on the page

<img width="637" alt="image" src="https://github.com/user-attachments/assets/eca72a24-eb88-4efa-bf01-194ded608095">